Frontend/android-native/app/src/driver/nammaYatriPartner/AndroidManifest.xml

```diff
@@ -48,6 +48,7 @@
                 <data android:scheme="http"/>
                 <data android:host="nammayatri.in"/>
                 <data android:path="/p" />
+                <data android:path="/refer" />
```

Comment:
This can be added in main AndroidManifeast

Reply:
no it can't be because of different domains

Reply:
What do you mean by different domains ?